sanic/request.py

```diff
@@ -7,7 +7,11 @@
 try:
     from ujson import loads as json_loads
 except ImportError:
-    from json import loads as json_loads
+    from json import loads as _json_loads
+
+    def json_loads(i):
+        i = i.decode() if isinstance(i, bytes) else i
```

Comment:
I think in the interest of speed here it'd be better off just doing a try, except. Try to decode if bytes, catch an `AttributeError` if the `decode` method does not exist. Similar to what we do in other places as well.